src/applications/vaos/reducers/expressCare.js

```diff
@@ -19,11 +22,13 @@ import { FETCH_STATUS } from '../utils/constants';
 
 const initialState = {
   windowsStatus: FETCH_STATUS.notStarted,
-  hasWindow: false,
-  allowRequests: false,
+  windows: null,
   localWindowString: null,
   minStart: null,
   maxEnd: null,
+  data: {},
```

Comment:
This is a minor thing but what do you think of putting all form state in a separate object? e.g:

```
...
maxEnd: null,
newRequest: {
  data: {},
  pages: {},
  submitStatus: FETCH_STATUS.notStarted
}
```
With appointments, we didn't have to worry about it since newAppointment and appointments are already separated. But with express care it might keep the state a little clearer.

Reply:
Yeah, that seems good to me. I was trying to do the minimum to hook up the submission, however you want to organize this works for me.